### PR TITLE
8253100: Fix "no comment" warnings in java.base/java.net

### PR DESCRIPTION
**Pull request by dfuch**
Hi, 

Please find here a change that fixes "no comment" warnings generated by `javadoc -Xdoclint` for `java.base/java.net`

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8253100](https://bugs.openjdk.java.net/browse/JDK-8253100): Fix "no comment" warnings in java.base/java.net

### Reviewers
 * [Rahul Yadav](https://openjdk.java.net/census#ryadav) (@rhyadav - Committer) ⚠️ Review applies to ff11d016ab2fcb9739c3695927f676f238919306
 * [Chris Hegarty](https://openjdk.java.net/census#chegar) (@ChrisHegarty - **Reviewer**) ⚠️ Review applies to ea605ca68dea42baa2890b39fad6400215fd7c40
 * [Naoto Sato](https://openjdk.java.net/census#naoto) (@naotoj - **Reviewer**) ⚠️ Review applies to ea605ca68dea42baa2890b39fad6400215fd7c40
 * [Alan Bateman](https://openjdk.java.net/census#alanb) (@AlanBateman - **Reviewer**) ⚠️ Review applies to ea605ca68dea42baa2890b39fad6400215fd7c40

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/2715/head:pull/2715`
`$ git checkout pull/2715`

### CODE REVIEW

File: src/java.base/share/classes/java/net/HttpRetryException.java [hunk 1]
```diff
@@ -39,7 +39,14 @@
     @java.io.Serial
     private static final long serialVersionUID = -9186022286469111381L;
 
+    /**
+     * The response code.
+     */
     private int responseCode;
+
+    /**
+     * the URL to be redirected to.
```
Comment (pconcannon) [inline src/java.base/share/classes/java/net/HttpRetryException.java:48]: Should this begin with a capital 't' like the previous comment above?

Comment (dfuch) [reply src/java.base/share/classes/java/net/HttpRetryException.java]: Good catch. Fixed.

**Comment by dfuch** [comment]
The CSR draft can be reviewed here:
https://bugs.openjdk.java.net/browse/JDK-8262372

File: src/java.base/share/classes/java/net/InetSocketAddress.java [hunk 2]
```diff
@@ -308,6 +321,10 @@ private void readObject(ObjectInputStream in)
         UNSAFE.putReference(this, FIELDS_OFFSET, h);
     }
 
+    /**
+     * Throws {@code InvalidObjectException}, always.
+     * @throws ObjectStreamException always
```
Comment (naotoj) [inline src/java.base/share/classes/java/net/InetSocketAddress.java:326]: Is this correct? Seems to contradict each other.

Comment (dfuch) [reply src/java.base/share/classes/java/net/InetSocketAddress.java:326]: `InvalidObjectException` is a subclass of `ObjectStreamException`.

**Comment by rhyadav** [comment]
LGTM!

**Review by rhyadav — APPROVED** [review]
LGTM!

**Comment by c-cleary** [comment]
Ran the changes through `-Xdoclint:missing` and there were no warnings outputted so looks good!

**Review by ChrisHegarty — APPROVED** [review]

**Review by naotoj — APPROVED** [review]

File: src/java.base/share/classes/java/net/UnixDomainSocketAddress.java [hunk 3]
```diff
@@ -95,6 +95,8 @@ private Object readResolve() {
      *
      * @return a {@link Ser}
      * representing the path name of this instance
```
Comment (AlanBateman) [inline src/java.base/share/classes/java/net/UnixDomainSocketAddress.java:97]: is there a spurious line break in the middle of the return description?

File: src/java.base/share/classes/java/net/URISyntaxException.java [hunk 4]
```diff
@@ -41,7 +41,14 @@
     @java.io.Serial
     private static final long serialVersionUID = 2137979680897488891L;
 
+    /**
+     * The input string.
+     */
     private String input;
+    /**
```
Comment (AlanBateman) [inline src/java.base/share/classes/java/net/URISyntaxException.java:48]: I'd probably put a blank line after input to space it out.

**Review by AlanBateman — APPROVED** [review]
Looks fine, just a few formatting nits.

**Comment by dfuch** [comment]
Thanks Alan. Fixed.
Would anyone review the CSR?
https://bugs.openjdk.java.net/browse/JDK-8262372

**Pull request closed without merging**